src/plugins/vis_augmenter/public/constants.ts

```diff
@@ -4,11 +4,10 @@
  */
 
 export const VIS_LAYER_COLUMN_TYPE = 'vis_layer';
-// TODO: replace with a value imported from OUI
```

Comment:
I believe so, I had removed it since with all of the concurrent OUI going on it's not needed for now. If it's decided later on we want more consistency (including the colors used in the regular chart data), we can modify.

Reply:
Opened https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4143